sunpy/timeseries/tests/test_timeseriesbase.py

```diff
@@ -708,6 +709,43 @@ def test_ts_sort_index(generic_ts):
     assert generic_ts.sort_index().to_dataframe().equals(generic_ts.to_dataframe().sort_index())
 
 
+def test_repr_html_(generic_ts):
+    html_string = generic_ts._repr_html_()
+    assert isinstance(html_string, str)
+
+
+def test_quicklook(generic_ts):
+    with mock.patch('webbrowser.open_new_tab') as mockwbopen:
```

Comment:
This seems like an easy switch, but I am not familiar with the syntax of pytest-mock and am finding its implementation somewhat puzzling at the moment. It might be best that I update it in a future PR to avoid holding up this one.